addons/survey/wizard/survey_invite.py

```diff
@@ -74,8 +74,7 @@ def _get_default_author(self):
 
     @api.depends('partner_ids', 'survey_id')
     def _compute_existing_partner_ids(self):
-        existing_answers = self.survey_id.user_input_ids
-        self.existing_partner_ids = existing_answers.mapped('partner_id') & self.partner_ids
+        self.existing_partner_ids = list(set(self.survey_id.user_input_ids.partner_id.ids) & set(self.partner_ids.ids))
```

Comment:
Would you mind writing a small python test for this?

I think you can simulate a NewId use case by using something like 

```py
with Form(self.env['survey.invite']) as invite_wizard:
    invite_wizard.partner_ids = yourexistingpartner
    # assert that invite_wizard.existing_partner_ids contains the partner
```

Can you give it a try?

Thanks!

Reply:
Thanks, it is a good idea, I have added it as a sub test of test_survey_invite, checking existing_partner_ids and the warning. The sub test fails without the fix.